sphinxcontrib/images.py

```diff
@@ -157,17 +157,16 @@ def run(self):
         if self.is_remote(self.arguments[0]):
             img['remote'] = True
             if download:
-                img['uri'] = os.path.join('_images', hashlib.sha1(self.arguments[0].encode()).hexdigest())
+                reluri = os.path.join(conf['cache_path'], hashlib.sha1(self.arguments[0].encode()).hexdigest())
+                img['uri'] = os.path.join('/', reluri) # relative to source dir
                 img['remote_uri'] = self.arguments[0]
-                env.remote_images[img['remote_uri']] = img['uri']
-                env.images.add_file('', img['uri'])
```

Comment:
I find that remote images with `:download: true` are downloaded to the custom cache directory but not copied to `_build/html/_images/` upon building. Hence they are not shown in the built HTML-docs. Adding `env.images.add_file()` back fixes this.

See also my comment here https://github.com/sphinx-contrib/images/pull/23#issuecomment-870083460